field_rrule/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Repetition rules
+================
+
+This module was written to offer a field type that holds rrules according to RFC 2445.
```

Comment:
Just to note - 5545 differences were mainly in implementation & error validation. While this could be something the library would handle, it's all easily avoided module-side too. IMO it would be advantageous to claim 5545, or at least note that this module attempts to conform - because it does currently.